src/uvw/resource.hpp

```diff
@@ -29,19 +29,19 @@ class Resource: public Emitter<T>, public std::enable_shared_from_this<T> {
           pLoop{std::move(ref)},
           resource{}
     {
-        this->template get<U>()->data = static_cast<T*>(this);
+        resource.data = static_cast<T*>(this);
     }
 
     auto parent() const noexcept {
         return pLoop->loop.get();
     }
 
-    template<typename R>
+    template<typename R = U>
```

Comment:
My original intent was to add another protected member, something like `getNative` (one `const` qualified overload, another not) that returns the raw pointer `U*` (or `const U*` accordingly). That would avoid the template instantiation altogether. But it involves changing the used `get` function in a lot of places.
Please tell me if that is a favorable option.